### PR TITLE
Implements form field copy button

### PR DESCRIPTION
**Pull request by sixlive**
## Description
Adds the copy button to the form view.

kapture

## Tasks
- [x] Tests
- [ ] Documentation
- [ ] Changlog Entry

## Related Issues/PRs
* Resolves https://github.com/sixlive/nova-text-copy-field/issues/20

## Steps to Verify Functionality
n/a

### CODE REVIEW

File: resources/js/components/CopyButton.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-    <div>
+    <div class="flex content-center">
         <div v-show="isNotDirty">
```
Comment (mikhailkozlov) [inline resources/js/components/CopyButton.vue:3]: This is great feature! Tested locally and it works great. Thank you. 

For some reason on my screen clipboard is all the way at the top of the line. 

image

Adding  `class="pt-2"` to `<div v-show="isNotDirty">` fixes that

image

Comment (sixlive) [reply resources/js/components/CopyButton.vue:3]: What browser are you using?

Comment (mikhailkozlov) [reply resources/js/components/CopyButton.vue:3]: Chrome Version 72.0.3626.119 (Official Build) (64-bit)

Also tested on Safari Version 12.0.3 (14606.4.5)

Comment (sixlive) [reply resources/js/components/CopyButton.vue:3]: I just pushed a fix to this branch, can you let me know if it fixed it for you? I'll merge after that.

**Pull request merged**